predicators/utils.py

```diff
@@ -2199,19 +2199,17 @@ def _stripped_classifier(state: State, objects: Sequence[Object]) -> bool:
 
 
 def strip_task(task: Task, included_predicates: Set[Predicate]) -> Task:
-    """Create a new task where any excluded predicates have their classifiers
-    removed."""
+    """Create a new task where any excluded goal predicates have their
+    classifiers removed."""
     stripped_goal: Set[GroundAtom] = set()
     for atom in task.goal:
-        # The atom's goal is known.
```

Comment:
Removed this comment because it is confusing. The goal is indeed known -- a more precise commen would say if the predicate in this atom was excluded or not. But the code itself says that pretty clearly, so any comment will be pretty redundant.